fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -2685,6 +2690,7 @@ public CommandProcessingResult chargeOff(JsonCommand command) {
             this.noteRepository.save(note);
         }
 
+        postJournalEntries(loan, existingTransactionIds, existingReversedTransactionIds);
         // TODO: add accounting
```

Comment:
you can remove this TODO line, accounting was added.

Reply:
Fixed